backend/api/management/commands/import_flights.py

```diff
@@ -0,0 +1,51 @@
+from django.core.management.base import BaseCommand
+from api.models import Flight
+import pandas as pd
+import os
+
+
+class Command(BaseCommand):
+    help = 'Import flight data from an Excel file and populate the database.'
+
+    def handle(self, *args, **options):
+        # Replace 'your_excel_file.xlsx' with the actual path to your Excel file.
+        data = pd.read_excel(r"C:\Users\Nitya\Desktop\ENGG\Capstone\GitHub\BabelBot-Code\backend\api\flight_details_with_airport.xlsx")
+        # data = pd.read_excel(os.path.join(os.path.dirname(__file__), "flight_details_with_airport.xlsx"))
```

Comment:
This is hard-coded right now, before we merge can we fix it so that it works on everyone's machines?